<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APRWidget</class>
 <widget class="QWidget" name="APRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add new record</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_name"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_unit">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edit_unit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_expiry">
        <property name="text">
         <string>Expiry</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edit_expiry"/>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="action_add">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="action_clear">
          <property name="text">
           <string>C&amp;lear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="action_cancel">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_name</tabstop>
  <tabstop>edit_unit</tabstop>
  <tabstop>edit_expiry</tabstop>
  <tabstop>action_add</tabstop>
  <tabstop>action_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
